<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesPageView</class>
 <widget class="QWidget" name="NotesPageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>516</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NotesPageView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame {
    background-color: #37474f;
border-top: none;
}
QLabel {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(55, 71, 79, 255), stop:0.5 rgba(38, 50, 56, 255), stop:1 rgba(34, 42, 48, 255));
    border: none;
    border-bottom: 1px solid #191919;
    color: #b0bec5;
    font: 600 16px &quot;Open Sans&quot;;
}

EntityWidget  {
  background-color: #37474f;
  border: none;
}
</string>
  </property>
  <widget class="QWidget" name="content" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>451</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>451</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
    background-color: #37474f;
    border-top: none;
    border-bottom: 1px solid #191919;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignLeft">
     <widget class="EntityWidget" name="subpage_notesAboutPlayers" native="true">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>225</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">EntityWidget {
    border: 1px solid #191919;
    border-left: none;
    border-top: 1px solid #455a64;
    border-top-right-radius: 1px 0px;
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1" rowspan="2">
        <widget class="NotesListEditor" name="notesListEdit" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>106</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#notesListEdit {
    border-right: none;
    border-top: none;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="title_notesAboutPlayers">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>33</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>33</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
border-top: none;
/*border-right: 1px solid #191919;
*/
}</string>
         </property>
         <property name="text">
          <string>@title_notesAboutPlayers</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="NotesList" name="notesList" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="EntityWidget" name="subpage_colorMarks" native="true">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>225</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">EntityWidget {
    border: 1px solid #455a64;
    border-right: none;
    border-bottom: 1px solid #191919;
    border-bottom-left-radius: 0px 1px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="title_colorMarks">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>33</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>33</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-top: none;</string>
         </property>
         <property name="text">
          <string>@title_colorMarks</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MarkersList" name="markers" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>55</red>
               <green>71</green>
               <blue>79</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="footer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>451</y>
     <width>960</width>
     <height>66</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>66</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>66</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #37474f;
    border: none;
    border-top: 1px solid #455a64;
    font: 13px &quot;Open Sans Semibold&quot;;
    color: #fafafa;
    padding: 10px 150px 10px 210px;
}</string>
   </property>
   <property name="text">
    <string>@text_notesInfo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NotesList</class>
   <extends>QWidget</extends>
   <header>noteslist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NotesListEditor</class>
   <extends>QWidget</extends>
   <header>noteslisteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MarkersList</class>
   <extends>QWidget</extends>
   <header>markerslist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>showNoteEditor(bool)</slot>
 </slots>
</ui>
